packages/trie/src/util/encoding.ts

```diff
@@ -67,19 +104,70 @@ export const compactBytesToNibbles = (compact: Uint8Array) => {
 }
 
 /**
- * A test helper to generates compact path for a subset of key bytes
+ * Packs every two nibbles into a single byte
  *
- * TODO: Commenting the code for now as this seems to be helper function
- * (from geth codebase )
+ * @param arr Nibble typed nibble array
+ * @returns Uint8Array typed byte array
+ */
+export const nibbleTypeToPackedBytes = (arr: Nibbles): Uint8Array => {
```

Comment:
some coverage tests would be nice here...

Reply:
Fixed